### PR TITLE
Add a nested query example

### PR DESCRIPTION
**Pull request by GGP1**
|Related issue|
|---|
| https://github.com/wazuh/wazuh/issues/16909 |

## Description

Adds a nested query example to the `User manual/RESTful API/Filtering data using queries` page.

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by mhamra — COMMENTED** [review]
The example is OK.

LGTM!

Thanks @GGP1

**Review by mhamra — APPROVED** [review]
The example is OK.

LGTM!

Thanks @GGP1

**Review by Selutario — APPROVED** [review]

**Comment by javimed** [comment]
The response output shows five instances for Ubuntu 18.04.4 LTS, two for Ubuntu 18.04.2 LTS, one for Windows (Windows 7), and none for an OS version 14. I think it makes it unnecessarily long and not as varied as it could be. What if we...

- Keep only two of the Ubuntu 18.04.4 agents findings
- Keep only one of the Ubuntu 18.04.2 agents findings
- Add an example for version 14 (or change to 22 for Ubuntu 22 or similar change)
- Add another Windows example with a different version

or similar approach to showcase the broader scope of the query?

File: source/user-manual/api/queries.rst [hunk 1]
```diff
@@ -251,3 +251,63 @@ Getting the ubuntu agents with id other than 0 and lower than 4, whose name cont
        "message": "All selected agents information was returned",
        "error": 0
     }
+
+Getting agents with an id higher than ``007`` that run on *Windows* or whose operating system major version is either 14 or 18:
```
Comment (javimed) [inline source/user-manual/api/queries.rst:255]:
```suggestion
Getting agents with an ID higher than ``007`` that run on *Windows* or whose operating system major version is either 14 or 18:

```

**Review by javimed — APPROVED** [review]

**Pull request merged**